decidim-admin/config/locales/ca.yml

```diff
@@ -31,6 +31,7 @@ ca:
         body: Cos
         subject: Assumpte
       organization:
+        admin_terms_of_use_body: Body for the admin terms of use
```

Comment:
@tramuntanal this shouldn't modify the 	ca.yml` file, but the `en.yml`!

Reply:
OMG! yes thanks!